File: library/src/test/java/de/westnordost/osmfeatures/TestUtils.java
package de.westnordost.osmfeatures;

import java.util.Arrays;
import java.util.Collection;
import java.util.List;

import static org.junit.Assert.assertTrue;

public class TestUtils {
    public static <T> void assertEqualsIgnoreOrder(Collection<T> a, Collection<T> b)
    {
        assertTrue(a.size() == b.size() && a.containsAll(b));
    }

    @SafeVarargs public static <T> List<T> listOf(T... items) { return Arrays.asList(items); }
}

File: library/src/commonTest/kotlin/osmfeatures/TestUtils.kt
package osmfeatures

import org.junit.Assert.assertTrue

object TestUtils {
    fun <T> assertEqualsIgnoreOrder(a: Collection<T>, b: Collection<T>) {
        assertTrue(a.size == b.size && a.containsAll(b))
    }

    @SafeVarargs
    fun <T> listOf(vararg items: T): List<T> {
        return items.asList()
    }
}
